plugins/woocommerce/src/Admin/Features/Blueprint/RestApi.php

```diff
@@ -300,18 +312,24 @@ public function import_step( \WP_REST_Request $request ) {
 		$result          = $step_importer->import();
 
 		return array(
-			'success'  => $result->is_success(),
-			'messages' => $result->get_messages(),
+			'success'      => $result->is_success(),
+			'messages'     => $result->get_messages(),
+			'sessionToken' => $session_token,
```

Comment:
I don't think its common practice to pass the `sessionToken` as part of the network request body, we should be including this in the response header instead.
I believe we did have to do something like this:
```
$response = new \WP_REST_Response(
	array(
		'success'  => $result->is_success(),
		'messages' => $result->get_messages(),
   )
);
$response->header( 'X-Blueprint-Import-Session', $session_token );
return $response;
```

Reply:
Thanks for the suggestion! I've updated the code to include the sessionToken in the response header in c5b0004af65266850c33aa9d3a6f853fcad83d5b